src/main/java/uk/gov/hmcts/reform/orgrolemapping/scheduler/Scheduler.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.orgrolemapping.scheduler;
+
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.orgrolemapping.domain.service.OrganisationService;
+import uk.gov.hmcts.reform.orgrolemapping.domain.service.ProfessionalUserService;
+
+@Service
+public class Scheduler {
+
+    private final OrganisationService organisationService;
+    private final ProfessionalUserService userService;
+
+    public Scheduler(OrganisationService organisationService, ProfessionalUserService userService) {
```

Comment:
I know there's currently no alerts for this process but would it be worth adding them anyway for consistency with the other changes we're making?